### DISCUSSION TITLE
The previous repo is replaced with a new fork, causing installation error

### DISCUSSION THREAD

**Discussion by meteorlxy** [cpisciotta/xcbeautify#298] [General]
Seems that @tuist has forked this repo back recently: https://github.com/tuist/xcbeautify/

The previous `tuist/xcbeautify` repo should be redirected to the `cpisciotta/xcbeautify` before. But with the new fork, the redirection stopped working, causing a 404 error:

<img width="1053" alt="image" src="https://github.com/user-attachments/assets/0d8fe981-3296-4e74-8419-f334dd850908">

Just complaining about this strange breakage 😮‍💨 cc @cpisciotta

---

For people who are still using cocoapods to install a legacy version of xcbeautify, try this:

```ruby
pod 'xcbeautify', :git => 'https://github.com/cpisciotta/xcbeautify.git', :tag => '0.17.0'
```

Reference: https://guides.cocoapods.org/using/the-podfile.html#from-a-podspec-in-the-root-of-a-library-repo

**Comment by cpisciotta** [comment]
@meteorlxy Thanks for reporting this issue. Tuist once maintained this repository, and their fork is now causing some URL redirect issues. 

I've contacted the Tuist maintainers and requested that they delete the fork until I resolve this issue. 

Ultimately, I'll need to update the podspec. It's very out-of-date, but – at a minimum – it should point to the correct URL.

Sorry for the inconvenience!

**Comment by pepicrft** [comment]
Apologies for that @meteorlxy. We were not aware of the dependence on the redirect. The fork is now deleted. Could you check if it works fine now, and close the issue in that case?
